demo/KevinDemo/Stories/Payment/PaymentView.swift

```diff
@@ -0,0 +1,186 @@
+                                            logoUrlString: charity.logo,
+                                            isSelected: viewModel.viewState.selectedCharity?.id == charity.id,
+                                            onTap: {
+                                                viewModel.viewState.selectedCharity = charity
+                                            }
+                                        )
+                                        .padding(.trailing, viewModel.viewState.charities.last?.id == charity.id ? 0.0 : 16.0)
+                                    }
+                                    
+                                    Spacer(minLength: 0.0)
+                                }
+                            }
+                            .frame(height: 51.0)
+                        }
+                        
+                        Group {
+                            Text("kevin_window_main_details_label".localized())
+                                .style(.sectionHeader)
+                                .padding(.top, 28)
+                                .padding(.bottom, 12)
+                            
+                            Text("kevin_window_main_email_label".localized())
+                                .style(.textFieldName)
+                                .padding(.top, 8.0)
+                            
+                            KevinTextField(
+                                text: $viewModel.viewState.email,
+                                type: .email
+                            )
+                            
+                            Text("kevin_window_main_amount_label".localized())
+                                .style(.textFieldName)
+                                .padding(.top, 20.0)
+                            
+                            ZStack(alignment: .trailing) {
+                                KevinTextField(
+                                    text: $viewModel.viewState.amountString,
+                                    type: .amount
+                                )
+                                
+                                Text("kevin_window_main_amount_field_currency".localized())
+                                    .style(.currencyHint)
+                                    .padding(.trailing, 14.0)
+                            }
+                        }
+                        
+                        KevinAgreementCheckMark(isAgreementChecked: $viewModel.viewState.isAgreementChecked)
+                        
+                        Button {
+                            hideKeyboard()
+                            viewModel.openPaymentTypeSelection()
+                        } label: {
+                            Text(String(
+                                format: "kevin_window_main_proceed_button".localized(),
+                                viewModel.viewState.amountString == "" ? "0.00" : viewModel.viewState.amountString,
```

Comment:
better check isEmpty and use formatter, In different regions there is different separators like "0.00" and "0,00"